address_book/internal/service/service.go

```diff
@@ -82,14 +89,14 @@ func (a *AddressBookService) DeleteContact(_ context.Context, in *pb.DeleteConta
 		}, nil
 	}
 
-	if _, ok := a.data.Load(in.Phone); !ok {
+	err := a.db.Delete(&models.Contact{}, in.Phone).Error
+
+	if err != nil {
 		return &pb.DeleteContactResponse{
-			Msg: "Contact not found",
+			Msg: "delete error",
```

Comment:
I assume, it makes sense to inform the user about the root cause of the error. Please don't hide the `err` in a message

Reply:
Done